src/main/resources/hudson/plugins/ansicolor/Messages.properties

```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-DisplayName = Color ANSI Console Output
+plugin_DisplayName=Color ANSI Console Output

```

Comment:
I am just not familiar with the whole findbugs business. Those are warnings though, right? I worry that this introduces some dependency that's not available at runtime or something like that?